hbase-zookeeper/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKUtil.java

```diff
@@ -1857,12 +1857,14 @@ public static String dump(ZKWatcher zkw) {
           sb.append("\n ").append(child);
         }
       }
-      sb.append("\nRegion server holding hbase:meta: "
-        + MetaTableLocator.getMetaRegionLocation(zkw));
+      sb.append("\nRegion server holding hbase:meta:");
+      sb.append("\n ").append(MetaTableLocator.getMetaRegionLocation(zkw));
       int numMetaReplicas = zkw.getMetaReplicaNodes().size();
-      for (int i = 1; i < numMetaReplicas; i++) {
-        sb.append("\nRegion server holding hbase:meta, replicaId " + i + " "
-                    + MetaTableLocator.getMetaRegionLocation(zkw, i));
+      if (numMetaReplicas > 1) {
```

Comment:
nit: This extra check is redundant.

Reply:
> nit: This extra check is redundant.

Thanks @wchevreuil for your review and pointing out the problem. I fixed it.